cfgov/v1/jinja2/v1/events/_macros.html

```diff
@@ -128,27 +125,13 @@ <h2 class="event-venue__heading">
                         video_id=event.live_video_id,
                         button_pos='bottom-right'
                     ) }}
-                {% elif event_state == 'past' %}
-                    {% if event.archive_video_id %}
-                        {# DISPLAY VIDEO RECORDING #}
-                        {% import 'v1/includes/organisms/video-player.html' as video_player with context %}
-                        {{ video_player.render(
-                            video_id=event.archive_video_id,
-                            button_pos='bottom-right'
-                        ) }}
-                    {% elif event.post_event_image_type == 'image' %}
-                        {# DISPLAY SPECIFICALLY-CHOSEN POST-EVENT IMAGE #}
-                        {% set img = image(event.post_event_image, 'width-1416') %}
-                        <img class="o-post-preview__image"
-                             src="{{ img.url }}"
-                             alt="{{ image_alt_value(img) }}">
-                    {% else %}
-                        {# DISPLAY PLACEHOLDER POST-EVENT IMAGE #}
-                        {% set img = image(event.post_event_image, 'width-1416') %}
-                        <img class="o-post-preview__image"
-                             src="{{ static('img/cfpb_video_cover_card_1380x776.png') }}"
-                             alt="">
-                    {% endif %}
+                {% elif event_state == 'past' and event.archive_video_id %}
+                    {# DISPLAY VIDEO RECORDING #}
+                    {% import 'v1/includes/organisms/video-player.html' as video_player with context %}
+                    {{ video_player.render(
+                        video_id=event.archive_video_id,
+                        button_pos='bottom-right'
+                    ) }}
```

Comment:
With this change, the placeholder image will never display after an event is over. I assume that's the large CFPB logo you referenced? 

Right now it looks like we have 96 events where the `post_event_image_type` is `"placeholder"` and `archive_video_id` is null that will show this, i.e. https://www.consumerfinance.gov/about-us/events/archive-past-events/field-hearing-on-medical-debt-collection/

<img width="788" alt="image" src="https://github.com/user-attachments/assets/e8b946cf-b73c-407c-9df9-d66a4a6d7e46">

Reply:
Yeah, the idea is to remove the "placeholder" altogether and only show it if it's explicitly added (now via the `venue_image` field).